src/commands/automercy.tsx

```diff
@@ -24,6 +36,7 @@ export const autoMercyCmd = createCommand(
     } else {
       inter.reply("autoMercy is " + autoMercy);
     }
+    updateSite(<WebAutoMercyDisplay />);
```

Comment:
Whenever you want to update the site you call this function from anywhere. It will render the component and replace the one on the website that has the same `id` attribute. In this case it will replace the div with the "status" attribute